R/inline_text.R

```diff
@@ -264,19 +277,16 @@ inline_text.tbl_survival <-
 
     }
 
-    # select timepoint --------------------------------------------------------
-    # get timepoint to display in result
-    all_times <- x$table_body %>%
-      select(time) %>%
-      pull()
+    # select time ----------------------------======----------------------------
+    # get time to display in result
+    all_times <- x$table_body$time
```

Comment:
just shortened this up a bit.  also, the `select(time)` causes a warning during CMD R Check, because of the bare reference to `time`